gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14-bin.zip
```

Comment:
Is this right? Only this number here changed but not the gradle wrapper itself?

Reply:
Do we even need to update gradle for this?

Reply:
Yes, loom 1.11 requires it.

Reply:
Do we need to upgrade loom? What changed?

Reply:
Alright then just need to update Gradle properly in the PR

Reply:
When you run it, it also changes the `gradlew` scripts, which are part of the repository. When you don't change it and only change `gradle/wrapper/gradle-wrapper.properties`, only change the wrapper itself and not the script. If there was a bug or vulnerability in the script, it would stay there, even if it was fixed in a newer gradle version. Running `gradlew wrapper --gradle-version 8.14` will also update the whole script.